src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/config/ApplicationConfig.java

```diff
@@ -4,20 +4,29 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 import uk.gov.companieshouse.certifiedcopies.orders.api.interceptor.LoggingInterceptor;
+import uk.gov.companieshouse.certifiedcopies.orders.api.interceptor.UserAuthenticationInterceptor;
+import uk.gov.companieshouse.certifiedcopies.orders.api.interceptor.UserAuthorisationInterceptor;
+import uk.gov.companieshouse.certifiedcopies.orders.api.service.CertifiedCopyItemService;
 
 import static com.fasterxml.jackson.databind.PropertyNamingStrategy.SNAKE_CASE;
 
 @Configuration
 public class ApplicationConfig implements WebMvcConfigurer {
 
+    @Autowired
+    private CertifiedCopyItemService certifiedCopyItemService;
+
     @Override
     public void addInterceptors(final InterceptorRegistry registry) {
         registry.addInterceptor(new LoggingInterceptor());
+        registry.addInterceptor(new UserAuthenticationInterceptor()).addPathPatterns("/orderable/**");
```

Comment:
This can probably be /orderable/certified-copies/** too

Reply:
Thanks. Fixed